framework/meta/src/cmd/standalone/template/copy_util.rs

```diff
@@ -1,17 +1,28 @@
 use std::{
-    fs::{self},
+    fs::{self, OpenOptions},
+    io::{BufWriter, Write},
     path::{Path, PathBuf},
 };
 
+use crate::version_history::validate_template_tag;
+
+const VERSION_0_44_0: usize = 39;
+
 /// Will copy an entire folder according to a whitelist of allowed paths.
 ///
 /// The whitelist is expected to contain paths relative from the source path.
 ///
 /// If a folder is whitelisted, then everything in the folder is considered whitelisted too.
 ///
 /// The function creates all necessary folder structure in the target, no additional preparation required.
-pub fn whitelisted_deep_copy(source_root: &Path, target_root: &Path, whitelist: &[String]) {
+pub fn whitelisted_deep_copy(
+    source_root: &Path,
+    target_root: &Path,
+    whitelist: &[String],
+    args_tag: &str,
+) {
     perform_file_copy(source_root, &PathBuf::new(), target_root, whitelist);
+    create_multiversx_json_file(target_root, args_tag);
```

Comment:
This is overkill, te file exists in the old versions too, you just need to add its name to the whitelist.